### PR TITLE
change asset and bg

### PR DESCRIPTION
**Pull request by piensaenpixel**
Hey @saleiva, @carlostallon, @mariodelvalle  and @matallo  take a look these changes:
- Update the brand, and change the blue.

image

### CODE REVIEW

File: _scss/main.scss [hunk 1]
```diff
@@ -61,27 +61,30 @@
...
-  margin-top: -4px;
 }
 
 .brand {
   display: inline-block;
   vertical-align: middle;
   zoom: 1;
-  width: 122px;
-  height: 43px;
+  width: 36px;
```
Comment (matallo) [inline _scss/main.scss]: new brand is `W: 140px | H: 36px`

File: _includes/navigation.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <header class="navbar">
   <div class="nav-header">
     <p class="nav-brand">
-      <a href="{{ '/' | prepend: site.baseurl }}" class="brand"><span>CartoDB</span></a>
+      <a href="{{ site.cartodb-baseurl }}" class="brand"><span>CartoDB</span></a>
```
Comment (matallo) [inline _includes/navigation.html]: link to CartoDB home?

Comment (piensaenpixel) [inline _includes/navigation.html]: Yes

File: _scss/main.scss (hunk 1 above)
Comment (piensaenpixel) [inline _scss/main.scss]: We are only showing the cartofante

Comment (piensaenpixel) [inline _scss/main.scss]: But i've just change the asset :dancer:

**Comment by matallo** [comment]
LGTM

still for me it's weird we link the brand with _website_

**Pull request merged**